src/PublicKey.h

```diff
@@ -30,6 +31,9 @@ class PublicKey {
     /// The number of bytes in a secp256k1 extended public key.
     static const size_t secp256k1ExtendedSize = 65;
 
+    /// The number of bytes in a nist256p1 public key.
+    static const size_t nist256p1Size = 33;
```

Comment:
@alejandro-isaza what is the proper way to handle for `nist256p1Size` here:
https://github.com/TrustWallet/wallet-core/blob/243c4d313db2dea6c9bab1d6ee240f62bde1d0b8/src/PublicKey.h#L41-L75

and here 
https://github.com/TrustWallet/wallet-core/blob/243c4d313db2dea6c9bab1d6ee240f62bde1d0b8/src/PublicKey.h#L60-L74

Reply:
nist and secp256 are pretty similar so we can may just have `ecdsaSize` for both.

Reply:
👌